assets/styles/components/media/_interactive-content.scss

```
@@ -32,6 +32,11 @@
     margin-bottom: 0;
     text-align: left;
   }
+
+  .embed div.figcaption { 
```

@dannylonglegs This isn't quite right, see: https://github.com/pressbooks/pressbooks/blob/dev/inc/shortcodes/complex/class-complex.php#L215

It should be:

```suggestion
  .embed figcaption {
```

---

it didnt target properly without the 'div' if i remember correctly. ill test again